packages/serverpod_serialization/lib/src/copy_with.dart

```diff
@@ -59,7 +39,12 @@ const _noneMutableTypes = [
   UuidValue,
 ];
 
-dynamic _guardedCopyWith(dynamic element) {
+/// Keyword helper for the `cloneTrivial` function
+const cloneTrivialFunctionName = 'cloneTrivial';
```

Comment:
Any ideas for a better name? Maybe `cloneNonNestedTypes` is more clear? Not sure what feels most intuitive!

Reply:
Good suggestion, moved to the private `Keywords` helper 👍